docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -56,7 +82,9 @@ const theme = createTheme({
 });
 ```
 
-### Extended `@date-io` adapters
+## Date library and adapters
+
+### Do not import adapter from `@date-io`
```

Comment:
Hm... For other changes we are listing what has been done and for this one you have changed to say what you should no longer do on v6. 🤔 
Maybe let's stick to the same style of listing what has been changed for consistency?

Reply:
Would you replace it with something like 

```suggestion
### Import the adapter from `@mui/x` packages
```

Reply:
Maybe something more like this (closer to original)?
```suggestion
### Require extended `@date-io` adapters
```

Reply:
My problem was I don't think `extended` is understandable outside of MUI.
I don't think people should care if the adapter we export is a wrapper from `@date-io` or just a new implementation.

Reply:
Understood and agreed. 👌 
That's why I believe my initial section heading was something like __Drop direct `@date-io` adapters usage__ 🤔

Reply:
It was `Extended `@date-io` adapters`

> or other changes we are listing what has been done and for this one you have changed to say what you should no longer do on v6

By the way, some of my title are probably bad, because I tried to do the exact opposite :laughing: 

On the core, the titles represent what people have to do in order to migrate

Reply:
> t was `Extended `@date-io` adapters`

By initial I wanted to say the original suggestion from me when I opened PR, but we changed it during code review. 🙈 

> On the core, the titles represent what people have to do in order to migrate

Hm... Good point. I'd say it makes sense then to change the headings to be more in line with mui core. What do you thin @joserodolfofreitas ?

Reply:
What about

> Replace `date-io` adapters with `@mui` ones

Reply:
It would be fine for me